tests/system/CLI/CLITest.php

```diff
@@ -665,4 +665,21 @@ public function testStrlen(): void
         $this->assertSame(7, CLI::strlen(CLI::color('success', 'green')));
         $this->assertSame(0, CLI::strlen(null));
     }
+
+    public function testPrintKeysAndValues(): void
+    {
+        $options = [
+            'c' => 'Enable only config caching.',
+            'l' => 'Enable only locator caching.',
+            'd' => 'Disable config and locator caching.',
+        ];
+
+        CLI::printKeysAndValues($options);
+
+        $expected = '  [-c]  Enable only config caching.' . PHP_EOL;
+        $expected .= '  [-l]  Enable only locator caching.' . PHP_EOL;
+        $expected .= '  [-d]  Disable config and locator caching.' . PHP_EOL;
+
+        $this->assertSame($this->getStreamFilterBuffer(), $expected);
```

Comment:
The method outputs the options (format like `[-c]`) and the values (descriptions).
It seems it is specialized only for command option output.
So the name `printKeysAndValues` is too general. I think more specific name is better.
E.g.,  `printOptionsDescription()`?

Reply:
No. The test code is not correct.
The method is now used in `promptByKey()`. It is used to display options for user input.
It is not for command option.

Reply:
`printOptionsDescription()` isn't too bad, but the name is a bit long.

how about `printOptionsList()` ?